backend/python/app/graphql/schema.py

```diff
@@ -51,6 +54,10 @@ class Query(graphene.ObjectType):
         user_id=graphene.Int(),
         translator=graphene.Boolean(),
     )
+    story_translation_by_id = graphene.Field(
+        graphene.List(StoryTranslationResponseDTO),
```

Comment:
this resolver doesnt work - the return type needs to be `StoryTranslationResponseDTO` not `graphene.List(StoryTranslationResponseDTO)`